_config.yml

```diff
@@ -59,7 +59,8 @@ sphinx:
       launch_buttons:
         binderhub_url: http://binder.mypythia.org
         notebook_interface: jupyterlab
-      extra_navbar: |
+      extra_navbar: ""
+      extra_footer: |
         Theme by <a href="https://projectpythia.org">Project Pythia</a>.<br><br>
```

Comment:
I think this is a great idea to simplify the user-facing content!

I suggest removing the line breaks so "Theme by..." and the licensing info all appear on a single line, as is currently the case [in Pythia Foundations](https://foundations.projectpythia.org/landing-page.html)